apps/ledger-live-mobile/src/screens/Manager/AppsList/AppRow.tsx

```diff
@@ -99,10 +92,22 @@ export default memo(function AppRow({
                 />
               </>
             )}
-          </Text>
-        </VersionContainer>
+          </Tag>
+          {authorName && displayAppDeveloperName ? (
```

Comment:
How is `conditions && <Component />` which is more simpler and has the same behaviour ?

Reply:
if `authorName` is `""`, `authorName && whatever && <Component />` will eval to `""` and cause a rendering error because `<Flex />` cannot take a `string` as a children.

Reply:
I can however do this:
```tsx
{Boolean(authorName) && displayAppDeveloperName &&  <Text />}
```

Reply:
But in general I prefer `condition ? <Component /> : null` because it works regardless of whatever type condition is